web_poet/rules.py

```diff
@@ -0,0 +1,338 @@
+    ``web-poet`` already provides a default Registry named ``default_registry``
+    for convenience. It can be directly accessed via:
+
+    .. code-block:: python
+
+        from web_poet import handle_urls, default_registry, WebPage
+
+        @handle_urls("example.com", instead_of=ProductPageObject)
+        class ExampleComProductPage(WebPage):
+            ...
+
+        override_rules = default_registry.get_rules()
+
+    Notice that the ``@handle_urls`` decorator that we're using is a part of the
+    ``default_registry``. This provides a shorter and quicker way to interact
+    with the built-in default :class:`~.PageObjectRegistry` instead of writing
+    the longer ``@default_registry.handle_urls``.
+
+    .. note::
+
+        It is encouraged to simply use and import the already existing registry
+        via ``from web_poet import default_registry`` instead of creating your
+        own :class:`~.PageObjectRegistry` instance. Using multiple registries
+        would be unwieldy in most cases.
+
+        However, it might be applicable in certain scenarios like storing custom
+        rules to separate it from the ``default_registry``. This :ref:`example
+        <overrides-custom-registry>` from the tutorial section may provide some
+        context.
+    """
+
+    @classmethod
+    def from_apply_rules(
+        cls: Type[PageObjectRegistryTV], rules: List[ApplyRule]
+    ) -> PageObjectRegistryTV:
+        """An alternative constructor for creating a :class:`~.PageObjectRegistry`
+        instance by accepting a list of :class:`~.ApplyRule`.
+
+        This is useful in cases wherein you need to store some selected rules
+        from multiple external packages. See this :ref:`example
+        <overrides-custom-registry>`.
+        """
+        return cls({rule.use: rule for rule in rules})
+
+    @classmethod
+    def from_override_rules(
+        cls: Type[PageObjectRegistryTV], rules: List[ApplyRule]
+    ) -> PageObjectRegistryTV:
+        """Deprecated. Use :meth:`~.PageObjectRegistry.from_apply_rules` instead."""
+        msg = (
+            "The 'from_override_rules' method is deprecated. "
+            "Use 'from_apply_rules' instead."
+        )
+        warnings.warn(msg, DeprecationWarning, stacklevel=2)
+        return cls.from_apply_rules(rules)
+
+    def handle_urls(
+        self,
+        include: Strings,
+        *,
+        overrides: Optional[Type[ItemPage]] = None,
+        instead_of: Optional[Type[ItemPage]] = None,
+        to_return: Optional[Type] = None,
+        exclude: Optional[Strings] = None,
+        priority: int = 500,
+        **kwargs,
+    ):
+        """
+        Class decorator that indicates that the decorated Page Object should be
+        used instead of the overridden one for a particular set the URLs.
+
+        The Page Object that is **overridden** is declared using the ``instead_of``
+        parameter.
+
+        The Item Class could also be **overridden** using the ``to_return``
```

Comment:
I'd not call it "overridden"; it's actually overridden if to_return is passed, but it's a very different type of override - to_return argument overrides the item type defined in the page object class. Using the same terminology could be a bit confusing.

Reply:
Updated in https://github.com/scrapinghub/web-poet/pull/84/commits/fc0ba50610cfc78c9c463e7eab52b20a365c3985